src/layout/layout.js

```diff
@@ -328,6 +420,10 @@
         this.tabBar_ = this.header_.querySelector('.' + this.CssClasses_.TAB_BAR);
       }
 
+      if (this.drawer_) {
+        this.drawerMenuItems_ = this.drawer_.querySelectorAll('.mdl-navigation__link');
```

Comment:
Navigation links aren't the only thing able to be in a drawer. This behavior needs to support *all* focusable elements. This includes, `button`, `input`, `textarea`, etc. Otherwise certain parts can be focused even when the drawer(s) are hidden causing odd tabbing behavior.